igv_remote/_core.py

```diff
@@ -40,9 +54,10 @@ def _set_saveopts(self, img_dir, img_basename, img_init_id=0) :
         self._img_basename = img_basename
         self._img_id = img_init_id
     
-    def _set_viewopts(self, squish, collapse, viewaspairs, sort):
-        self._squish = squish
-        self._collapse = collapse
+    def _set_viewopts(self, view_type, viewaspairs, sort):
+        if view_type not in ['squished', 'collapsed', 'expanded']:
+            raise Exception("view_type must be one of [squished, collapsed, expanded]")
```

Comment:
Change generic `Exception` to `ValueError`. It's good to be precise with exceptions.